CredentialProvider.Microsoft/CredentialProviderArgs.cs

```diff
@@ -19,9 +19,14 @@ internal class CredentialProviderArgs
         [ArgDescription("If present and true, providers will not issue interactive prompts")]
         public bool NonInteractive { get; set; }
 
-        [ArgDescription("Notifies the provider that this is a retry and the credentials were rejected on a previous attempt")]
+        [ArgDescription("Always generate fresh credentials, ignoring the cache")]
+        [OmitFromUsageDocs] // IsRetry support is really for the NuGet flow and is confusing for standalone. Keep for backcompat, but don't advertise.
         public bool IsRetry { get; set; }
 
+        [ArgDescription("Skip validating cached credentials. Expired or revoked credentials may be returned.")]
+        [ArgShortcut("S")]
+        public bool SkipValidatingCachedCreds { get; set; }
```

Comment:
Thoughts on not even adding this option and just always validating? Even if someone is already using the IsRetry flow, the one extra GET isn't really significant overhead.

Reply:
I agree to nuke it